lookup-table.yaml

```diff
@@ -212,3 +212,15 @@
     - utility
     - library
     - api
+- id: lambdabettergrass
+  names:
+    - lambdabettergrass
+  tags:
+    - decoration
+    - cosmetic
+    - adventure-rpg
```

Comment:
Split into
```
  - adventure
  - rpg
```

Reply:
I wrote it as adventure-rpg because that's how it's listed on curseforge. Though I do agree that splitting it up would be a good idea, it helps readability slightly. Just as long as it stays consistent it should be fine.